### PR TITLE
Add Envoy dashboard documentation

### PR DESCRIPTION
**Pull request by jotak**
Closes https://github.com/kiali/kiali/issues/1820

### CODE REVIEW

**Comment by jotak** [comment]
Native-English speakers review welcome :)

File: content/documentation/runtimes-monitoring/index.adoc [hunk 1]
```diff
@@ -103,6 +103,17 @@ As an example and for self-monitoring purpose Kiali itself link:https://github.c
 
 The pod annotation for Kiali is: `kiali.io/runtimes: go`
 
+
+=== Envoy
+
+The Envoy sidecars that come with Istio are also supplying link:https://www.envoyproxy.io/docs/envoy/latest/configuration/upstream/cluster_manager/cluster_stats[some internal metrics], that you can see in Kiali. They are not the same than the metrics reported by Istio Telemetry, and of which Kiali makes an intensive usage. Some of them can be redundant.
```
Comment (jshaughn) [inline content/documentation/runtimes-monitoring/index.adoc]: This is not bad, here are two options:
Required change: 
`"not the same than" -> "not the same as"`
Optional change:
`Istio's Envoy sidecars supply <link>[some internal metrics] that can be viewed in in Kiali.  They are different than the metrics reported by Istio Telemetry, of which Kiali uses intensively.  Some of Envoy's metrics may be redundant.`

File: content/documentation/runtimes-monitoring/index.adoc [hunk 2]
```diff
@@ -103,6 +103,17 @@ As an example and for self-monitoring purpose Kiali itself link:https://github.c
...
+
+Unlike the other Runtimes dashboards, this one has to be enabled explicitely with the pod annotation `kiali.io/runtimes: envoy` (there's no automatic discovery configured).
```
Comment (jshaughn) [inline content/documentation/runtimes-monitoring/index.adoc]: Required change:
`explicitely -> explicitly`
Optional change:
`"has to be enabled explicitely with the pod" -> "needs to be explicitly enabled with pod"`

File: content/documentation/runtimes-monitoring/index.adoc (hunk 1 above)
Comment (JStickler) [reply content/documentation/runtimes-monitoring/index.adoc]: They are different than the metrics reported by Istio Telemetry, of which Kiali uses intensively.
I would revise as
They are different than the metrics reported by Istio Telemetry, which Kiali uses extensively.

File: content/documentation/runtimes-monitoring/index.adoc (hunk 2 above)
Comment (JStickler) [reply content/documentation/runtimes-monitoring/index.adoc]: Unlike the other Runtimes dashboards, there is no automatic discovery configured for Envoy.  You must explicitly enable the Envoy dashboard  with the pod annotation `kiali.io/runtimes: envoy`.

File: content/documentation/runtimes-monitoring/index.adoc [hunk 3]
```diff
@@ -103,6 +103,17 @@ As an example and for self-monitoring purpose Kiali itself link:https://github.c
...
+
+Note that the enabled Envoy metrics can be tuned, as explained in link:https://istio.io/docs/ops/telemetry/envoy-stats/[Istio documentation]: it's possible to get more metrics using the `statsInclusionPrefixes` annotation.
```
Comment (JStickler) [inline content/documentation/runtimes-monitoring/index.adoc]: as explained in **the** Istio documentation

File: content/documentation/runtimes-monitoring/index.adoc [hunk 4]
```diff
@@ -103,6 +103,17 @@ As an example and for self-monitoring purpose Kiali itself link:https://github.c
...
+
+E.g. `sidecar.istio.io/statsInclusionPrefixes: cluster_manager,listener_manager,listener` will add `listener` metrics to the usual ones, for more inbound traffic information. You can then customize the Envoy dashboard of Kiali according to the collected metrics.
```
Comment (JStickler) [inline content/documentation/runtimes-monitoring/index.adoc]: Replace "E.g." with "For example"

Also this example needs a bit more information, do you need all three (cluster_manager,listener_manager,listener) to add listener metrics?  The Istio docs say "To gather data on inbound traffic, add the listener prefix.", but without clicking over there, I would not have known that the other two settings were defaults.

**Comment by jotak** [comment]
Thanks @jshaughn @JStickler , updated

**Pull request merged**